executor/aggfuncs/func_avg_test.go

```diff
@@ -32,11 +32,11 @@ func (s *testSuite) TestMergePartialResult4AvgDecimal(c *C) {
 	iter := chunk.NewIterator4Chunk(srcChk)
 
 	desc := &aggregation.AggFuncDesc{
```

Comment:
Will it be better to define a function like `NewAggFuncDesc`?

Reply:
Same with you, what about `NewAggFuncDesc(Name, Args, RetTp, Mode)`?